apps/client/src/features/app-settings/panel/client-control-panel/ClientList.tsx

```diff
@@ -0,0 +1,205 @@
+import { useState } from 'react';
+import {
+  Badge,
+  Button,
+  ButtonGroup,
+  IconButton,
+  Input,
+  InputGroup,
+  InputLeftAddon,
+  Modal,
+  ModalBody,
+  ModalCloseButton,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  ModalOverlay,
+  useDisclosure,
+} from '@chakra-ui/react';
+import { IoClose } from '@react-icons/all-files/io5/IoClose';
+import { Clients } from 'ontime-types';
+
+import { setClientRemote } from '../../../../common/hooks/useSocket';
+import { useClientStore } from '../../../../common/stores/clientStore';
+import * as Panel from '../PanelUtils';
+
+import style from './ClientControlPanel.module.scss';
+
+export default function ClientList() {
+  const { id: myId, clients } = useClientStore();
+  const { isOpen: isOpenRedirect, onOpen: onOpenRedirect, onClose: onCloseRedirect } = useDisclosure();
+  const { isOpen: isOpenRename, onOpen: onOpenRename, onClose: onCloseRename } = useDisclosure();
+  const { setIdentify, setRedirect, setRename } = setClientRemote;
+
+  const [targetId, setTargetId] = useState('');
+
+  const openRename = (id: string) => {
```

Comment:
would it be best to call this something else to avoid shadowing the `id` from `useClientStore`? in that case we wouldnt need to rename the `id` to `myId`